lib/plugins/aws/package/compile/events/apiGateway/lib/usagePlan.js

```diff
@@ -31,10 +31,7 @@ function createUsagePlanResource(that, name) {
       that.serverless.service.service
     } ${that.provider.getStage()} stage`;
     // assign quota
-    if (
-      _.has(that.serverless.service.provider, 'usagePlan.quota') &&
-      that.serverless.service.provider.usagePlan.quota !== null
-    ) {
+    if (_.get(that.serverless.service.provider, 'usagePlan.quota', null) !== null) {
```

Comment:
Let's simplify it, to simply: `if (_.get(that.serverless.service.provider, 'usagePlan.quota')`

Note we're expecting object as `quota`, hence we should accept only truthy values